### DISCUSSION TITLE
Firebeetle 2 bricked?

### DISCUSSION THREAD

**Discussion by icho40** [arendst/Tasmota#22010] [Q&A] [answered]
Hi,
I'm not sure whether I've bricked my Firebeetle 2 or not..
After a flash process was aborted with an error,
my ESP32 no longer boots.
I could read this in the serial interface:

ets_main.c 331 
ESP-ROM:esp32c6-20220919
Build:Sep 19 2022
rst:0x7 (TG0_WDT_HPSYS),boot:0x68 (SPI_FAST_FLASH_BOOT)
Saved PC:0x40018bea
SPIWP:0xee
mode:DIO, clock div:2
load:0x42150020,len:0x90fc0
load:0x40800000,len:0xd9c4
load:0x4080d9d0,len:0x1664
load:0x42000020,len:0x1491ac
Invalid image block, can't boot.

Now I can neither reflash via USB nor with UART.
I have tried both with esptool and with Arduino.

What can I do?

**Comment by sfromis** [comment]
Well, it does look like you need to do a wired flash, with ESP32-C6, you should be able to do it via USB. Normal protocol is to bring it to flashing mode by holding the "boot" button while clicking the "reset" button before releasing the "boot" button again.

**Comment by icho40** [comment]
I have already tried it several times.

lsusb shows:
Espressif USB JTAG/serial debug unit

I have tried with USB and UART-USB.
but neither esptool nor Arduino recognize the ESP.
Also in the online flash tool from Tasmota
only the port 
“USB/JTAG/serial debug unit (ttyACM0) paired “repeatedly only briefly 

When trying to use Esptool I have just
first pressed “Boot”, then plugged the USB into the PC,
briefly pressed Reset and released it again.
With
esptool -p /dev/ttyACM0 chip_id

I got:
"esptool.py v2.8
Serial port /dev/ttyACM0
Connecting...
Detecting chip type...

A fatal error occurred: Unexpected UART datecode value 0x00000fff. Failed to autodetect chip type."

**Reply by barbudor** [reply, marked as answer]
your esptool looks to be largely outdated
mine is v4.7.0 and I haven't updated it for month
such old version may not known about C6

**Comment by icho40** [comment]
I think you're both right.
Thanks to sfromi's hint about the order of the boot and reset button and barbudo's hint about my outdated esptool version I was able to flash my Esp32-C6 again.

Many thanks for your help, I have learned something new.
